### ISSUE TITLE
Invoke-ScriptAnalyzer not found

### ISSUE THREAD

**Issue by ferrarimarco** [super-linter/super-linter#4771]
## Describe the bug

I got the following error when running super-linter:

```
Invoke-ScriptAnalyzer: The term 'Invoke-ScriptAnalyzer' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

Will provide more context when I investigate deeper.

Update (after local tests):

- This doesn't happen on super-linter v5.4.0.
- It started happening on v5.4.1, and happens on v.5.4.2 and v5.4.3.

Update (after checking the latest super-linter commits and CI jobs)

- Tests for the latest tag (v5.4.3) passed the Invoke-ScriptAnalyzer test ([ref](https://github.com/super-linter/super-linter/actions/runs/6549998298/job/17788143131#step:8:518)).

Update (after checking image sizes): slim images are smaller than non-slim ones, as expected. So it's probably unlikely that we messed up with the build pipeline when removing stuff when building slim images.

Update (after opening a shell in an instance running v5.4.3):

```shell
[redacted]:/# pwsh -c "Get-Module -Name PSScriptAnalyzer -ListAvailable -All" 2>&1
```

Doesn't report any output. So it appears that the `PSScriptAnalyzer` is not installed.

After installing it (`[redacted]:/# pwsh -c "Install-Module -Name PSScriptAnalyzer -RequiredVersion 1.21.0 -Scope AllUsers -Force"`:

```shell
[redacted]:/# pwsh -c "Get-Module -Name PSScriptAnalyzer -ListAvailable -All" 2>&1

    Directory: /usr/local/share/powershell/Modules

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Script     0.0                   PSScriptAnalyzer                    Desk
Script     1.21.0                PSScriptAnalyzer                    Desk      {Get-ScriptAnalyzerRule, Invoke-ScriptAnalyzer, Invoke-Formatter}
```

## Expected behavior

1. `Invoke-ScriptAnalyzer` should be available.
2. We should have a test case covering this issue (if we confirm that this is an issue!)

## Logs

See above.

Full run: https://github.com/ferrarimarco/dotfiles/actions/runs/6554723187/job/17822115827

**Comment by ferrarimarco** [comment]
Ah, I was on the right track. The current prod image (v5.4.3) fails the test suite:

```
  ×  super-linter-validate-powershell-modules: Super-Linter validate Powershell Modules (2 failed)
     ✔  Command: `pwsh -c "(Get-Module -Name PSScriptAnalyzer -ListAvailable | Select-Object -First 1).Name" 2>&1` exit_status is expected to eq 0
     ×  Command: `pwsh -c "(Get-Module -Name PSScriptAnalyzer -ListAvailable | Select-Object -First 1).Name" 2>&1` stdout is expected to eq "PSScriptAnalyzer\n"
     
     expected: "PSScriptAnalyzer\n"
          got: ""
     
     (compared using ==)
     
     Diff:
     @@ -1,2 +1 @@
     -PSScriptAnalyzer

     ✔  Command: `pwsh -c "(Get-Command Invoke-ScriptAnalyzer | Select-Object -First 1).Name" 2>&1` exit_status is expected to eq 0
     ×  Command: `pwsh -c "(Get-Command Invoke-ScriptAnalyzer | Select-Object -First 1).Name" 2>&1` stdout is expected to eq "Invoke-ScriptAnalyzer\n"
     
     expected: "Invoke-ScriptAnalyzer\n"
          got: "\e[31;1mGet-Command: \e[31;1mThe term 'Invoke-ScriptAnalyzer' is not recognized as a name of a cmdle...lling of the name, or if a path was included, verify that the path is correct and try again.\e[0m\n"
     
     (compared using ==)
     
     Diff:
     @@ -1,2 +1,3 @@
     -Invoke-ScriptAnalyzer
     +Get-Command: The term 'Invoke-ScriptAnalyzer' is not recognized as a name of a cmdlet, function, script file, or executable program.
     +Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

```

To reproduce:

```shell
docker pull ghcr.io/super-linter/super-linter:v5.4.3
CONTAINER_IMAGE_TARGET="standard" CONTAINER_IMAGE_ID="ghcr.io/super-linter/super-linter:v5.4.3"  make inspec
```

**Comment by smokedlinq** [comment]
@zkoppert possible to get the release published or an ETA when you think it'll be available?
